target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -744,7 +745,8 @@ ath79_setup_macs()
 	hak5,packet-squirrel)
 		label_mac=$(mtd_get_mac_binary u-boot 0x1fc00)
 		;;
-	huawei,ap5030dn)
+	huawei,ap5030dn|\
+	huawei,ap6010dn)
```

Comment:
erm this can be done with label-mac-device I believe.

Reply:
Works fine, thanks for the suggestion.